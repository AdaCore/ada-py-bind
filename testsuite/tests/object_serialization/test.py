import gen
import json

p = gen.from_json(json.dumps({"id": "lolwut", "x": 12, "y": 15}))
print p.id
print p.position.x
print p.position.y

p.position.x = 0
p.position.y = 2
p.id = "hahahoho"

print p.to_json()
