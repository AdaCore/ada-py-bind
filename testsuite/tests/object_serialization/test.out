lolwut
12
15
{"id":"hahahoho","x":0,"y":2}
